string name
time t
int32 trajectoryType
int32 direction
float64[] setpointRn
std_msgs/Float64MultiArray setpointSE3
float64[6] velocitySE3
bool maxCommandSpeed
